sql_alchemy_project8/project8/project8/test_sqlalchemy_proj8.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import sys
+import os
+from sqlalchemy_proj8 import validate_artist_df
+import pandas as pd
+import numpy as np
+from sqlalchemy_proj8 import validate_artwork_data
+from sqlalchemy_proj8 import create_database_engine
+
+
+def test_validate_artist_df():
+   
+    columns = ["DisplayName", "ArtistBio", "Nationality", "Gender", "BeginDate", "EndDate"]
+    data = {
```

Comment:
move data creation for your test to `conftest.py` file then use `pytest.fixtures`